src/platform/plugins/shared/discover/jest.config.js

```diff
@@ -16,4 +16,7 @@ module.exports = {
   collectCoverageFrom: [
     '<rootDir>/src/platform/plugins/shared/discover/{common,public,server}/**/*.{js,ts,tsx}',
   ],
+  globals: {
+    Uint8Array: Uint8Array,
+  },
```

Comment:
Do you know where we were seeing failures without this in Discover tests? Just curious since I'm surprised this package was called from our tests.

Reply:
Thanks for looking into it and reverting the change 🙂 You're correct that this is unrelated flakiness unfortunately. There's a failing test issue for it here, and we've been investigating it internally and will hopefully have a fix soon: #217005.